Makefile

```diff
@@ -1,14 +1,11 @@
 FAVICON=favicon
-LOGO=official-logo
+LOGO=logo
 
 all: gfx/favicon.svg gfx/favicon.ico gfx/logo.svg
 .PHONY: all
 
-iris-logo/%.pdf: iris-logo/%.tex
-	cd iris-logo && pdflatex $(notdir $<)
-
-iris-logo/%.svg: iris-logo/%.pdf
-	pdf2svg $< $@
+iris-logo/%.svg: iris-logo/%.tex
+	cd iris-logo && latexmk $(basename $(notdir $<))
```

Comment:
Should this have `-dvi`?

Reply:
ah yes, I missed there was a makefile sorry

Reply:
@bordaigorl this issue seems to be still open.

Reply:
Ah right the makefile.
Do we really want these files to be regenerated every make?
I can imagine it may cause spurious diffs just because one is compiling them on different systems getting different metadata and stuff...

Reply:
We could have them in the makefile but not in the default target.

Reply:
ok makes sense. on it